org.openjkdf2.OpenJKDF2.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.openjkdf2.OpenJKDF2
```

Comment:
openjkdf2.org is a placeholder. Is it own by the developer of that app?

Reply:
I'm the only developer of the app and the domain is mine, yes.